cfgov/jinja2/v1/_includes/molecules/image-inset.html

```diff
@@ -0,0 +1,74 @@
+{# ==========================================================================
+
+    Image Inset
+
+    ==========================================================================
+
+    Description:
+
+    Creates image inset markup when given:
+
+    value:                   Object defined from a StreamField block.
+
+    value.heading:           The heading of the image inset.
+
+    value.image:             A basic image object to insert.
+
+    value.image.upload:      The image object.
+
+    value.image.alt:         String for alt text for screenreaders.
+
+    value.paragraph:         String for text below the image.
+
+    value.is_image_left:     Boolean indicating whether to float the image
+                             to the left.
+
+    value.image_width:       String representing the image width.
+                             Possible values: (150, 270, 300).
+                             Defaults to 270.
+
+   ========================================================================== #}
+
+{# Image sizes are doubled for retina displays. Images created via Wagtail
+   are never stretched or upscaled and maintain their aspect ratio. #}
+{% set image_inset = image(value.image.upload, 'width-' ~ value.image_width
+                     | int * 2) %}
+{% set image_mobile = image(value.image.upload, 'width-1200') %}
+<div class="m-inset
+            m-inset__image
+            {{ 'm-inset__image__left' if value.is_image_left else '' }}
+            {{ 'm-inset__image__' ~ ( value.image_width
+               if value.image_width else '270' ) }}
+            {{ 'm-inset__image__portrait'
+               if image_inset.height > image_inset.width else 'landscape' }}">
+    {% if value.heading %}
+    <h2 class="header-slug">
+        <span class="header-slug_inner">
+            {{ value.heading }}
+        </span>
+    </h2>
+    {% endif %}
+    <div class="m-inset_image-container">
+        {% if value.is_image_decorative %}
```

Comment:
@schaferjh On a practical level, what it does is prevent screenreaders from reading out redundant or superfluous text. The vast majority of our illustrations fall into this category. Describing them aloud, or even announcing the presence of an image, is of no benefit to the user of a screenreader.